themes/orbit/layout/index.ejs

```diff
@@ -123,6 +153,8 @@
   Copyright &copy; <%- new Date().getFullYear() %> <a href="http://www.cerebris.com">Cerebris Corporation</a>.</p>
 </div>
 
+<!-- <div class="media-size-marker"><i class="number"></div> -->
```

Comment:
This is a widget I added in that displays the current screen size media query breakpoint in the lower left hand corner of the screen.  It can be helpful to see a visual of the breakpoint when adding responsiveness to the design.  

I'm not usually fond of leaving in commented out code, but I tend to make an exception for this on the sites I work on.